tools/run_tests/artifacts/package_targets.py

```diff
@@ -165,15 +165,23 @@ def build_jobspec(self, inner_jobs=None):
         dockerfile_dir = (
             "tools/dockerfile/grpc_artifact_python_manylinux2014_x64"
         )
+        environ = {
+            "PYTHON": "/opt/python/cp39-cp39/bin/python",
+            "ARTIFACT_PREFIX": "python_",
+            "EXCLUDE_PREFIX": "python_musllinux_1_1_aarch64_*",
```

Comment:
I think I'm going to take back my note on renaming it from `EXCLUDE_PATTERN`, unfortunately. My idea is that we'll have the same format as `ARTIFACT_PREFIX` and will remove `*` from here and put it into `find`. However, it'll complicate things drastically. Now when the var isn't set, find gets `-not -name ""` argument - which is a no-op, I checked. 

However, if we put a `*` to `-not -name "${EXCLUDE_PREFIX}*" \`, it'll resolve into `-not -name "*"` and will exclude everything. 

May I ask you to rename `EXCLUDE_PREFIX` back to `EXCLUDE_PATTERN`?